CONTRIBUTING.md

```diff
@@ -1,24 +1,62 @@
 # How to contribute to this project ?
 
-1. First you must choose an issue on [the project github](https://github.com/mdl29/scratchy). If you want to do something but no issue is created yet you must create one (try to tag it properly).
-2. Fork the project.
-3. Create a new branch whith the following name format on your fork : `resolves #ISSUEID Component: title for features/fix`.
+## **Choose a issue :**
 
-> Example : if you want to solve this issue (image below) you should have named your branch `resolvec #138 tool : kickstarter for yannis`.
-> Component are : backend / frontend / tools / doc
-> ![example issue](./doc/example_issue.png)
+You must choose an issue on [the project github](https://github.com/mdl29/scratchy). If you want to do something but no issue is created yet you must create one (try to tag it properly).
 
-4. When you think your code is ready to be merged to the project create a pull request, your code will be review and then you will be able to merge it yourself if it's accepted.
+## **Fork the project :**
 
-# What are the requirement for proposing a pull request ?
+You need to fork [scratchy repository](https://github.com/mdl29/scratchy)
 
-## There are the convention to respect for each programing language
+ * ℹ️ [how to fork a github project ?](https://guides.github.com/activities/forking/)
 
-| Language | Convention                                            |
-| -------- | ----------------------------------------------------- |
-| Python   | [PEP8](https://pep8.org/)                             |
-| YAML     | [the linter](https://github.com/adrienverge/yamllint) |
 
-## Other important notes
+## **Create your branch :**
 
-Try to comment your code in a clear way (we don't ask for a novel but someone who read your code must be able to tell what it does)
+ Create a new branch with the following name :
+ `ISSUEID-title_of_issue`
+  * **example :** `91-create_scratchy_service`
+  
+## **create a pull request :**  
+
+When you think your code is ready to be merged to the project create a pull request.
```

Comment:
add something like `if it's work in progress, set the [pull request status to draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/)`